src/fcu_io.cpp

```diff
@@ -82,6 +86,16 @@ void fcuIO::handle_mavlink_message(const mavlink_message_t &msg)
   case MAVLINK_MSG_ID_NAMED_VALUE_FLOAT:
     handle_named_value_float_msg(msg);
     break;
+  case MAVLINK_MSG_ID_SMALL_BARO:
+    handle_small_baro_msg(msg);
+    break;
+  case MAVLINK_MSG_ID_TIMESYNC: // <-- This is handled by the time_manager
+    break;
+  case MAVLINK_MSG_ID_PARAM_VALUE: // <-- This is handled by the param_manager
+    break;
+  default:
+    ROS_ERROR("fcu_io:Got unsupported message ID %d", msg.msgid);
```

Comment:
I'm just wondering if you're trying to accomplish something specific here? In my mind receiving an unhandled message isn't an error, it just means we're not interested in it. I had removed it at one point, but at the very least we should demote it to a warning.